toolkit/templates/layouts/_base_page.html

```diff
@@ -1,6 +1,13 @@
 {% extends "govuk/govuk_template.html" %}
 
 {% block head %}
+  {% with custom_dimensions=[
+        {"data_id": 11, "data_value": current_user.supplier_organisation_size }
+    ] if current_user and current_user.supplier_organisation_size else []
+  %}
+    {% include "toolkit/custom-dimensions.html" %}
+  {% endwith %}
+
   {% block custom_meta %}{% include "toolkit/custom-dimensions.html" %}{% endblock %}
```

Comment:
I'd rather avoid having two `custom-dimensions.html` includes in the base template if we can. 

Could we add an extra `{% with app_specific_custom_dimensions=[whatever] %}` so we can have our default ones here, and append any extra ones when we use the template in the app? 

Alternatively, if we're always going to set them here rather than in apps, why not do it in the `custom-dimensions.html` template itself, like we do with `current_user.role`?